packages/@uppy/utils/src/findDOMElement.ts

```diff
@@ -1,10 +1,10 @@
 import isDOMElement from './isDOMElement.ts'
 
-export default function findDOMElement<T>(
-  element: T,
+export default function findDOMElement<E, ReturnType = Element>(
+  element: E,
   context: Document = document,
-): T extends Element ? T
-: T extends Node | string ? Element | null
+): ReturnType extends Element ? ReturnType
+: ReturnType extends Node | string ? Element | null
```

Comment:
Could you move this to its own PR (along with the changes in `@uppy/core`) so we can discuss it separately?